src/index.ts

```diff
@@ -85,7 +81,20 @@ program
     )
     .action(async ({ name, category, majorVersion }) => {
         if (majorVersion) {
-            newMajorVersion();
+            inquirer
+                .prompt({
+                    default: true,
+                    message: `Are you sure you want to create a new major release? Make sure you have ways to update your current block version if you need to`,
```

Comment:
A little safe guard notice to remind devs to think about the version they are leaving behind. Open to better ways to express this.

Reply:
Maybe
```
message: `Are you sure you want to create a new major release? We recommend tagging your major releases and creating new branches from them for future updates.`,
```

Reply:
@chris-decker-volusion Updated the message, thanks.